examples/realtime-mc.yaml

```diff
@@ -16,6 +16,8 @@ spec:
       not_isolated_cores_expanded=${f:cpulist_invert:${isolated_cores_expanded}}
       [bootloader]
       cmdline_ocp_realtime=+systemd.cpu_affinity=${not_isolated_cores_expanded}
+      #[service]
+      #service.stalld=start,enable
```

Comment:
passing variables will be here ? (if not then a reference in the comment will suffice)

Reply:
Yes, I'm planning to document this.  A "pointer" to an environment file will be passed here.